mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
@mshamalainen and @staulu any idea why these point magneteter definitions
would differ between MNE and Elekta? It would be nice if we could reconcile
them.